src/components/RequestBlock/index.tsx

```diff
@@ -97,7 +97,6 @@ const RequestBlock: React.FC<RequestBlockProps> = ({ request, onUpdate }) => {
               <>
                 <Button
                   buttonType="success"
-                  buttonSize="sm"
```

Comment:
Why are we changing the button size here?

Reply:
It actually was `md` before, I think I changed it because the icons for these buttons don't match those of the `md` buttons in the rest of the UI. So this actually reverts the change you mentioned on Discord.

Reply:
Huh, i guess at some point I changed them back to `md`. I think they may have originally been smaller 🤷 Okay!

Reply:
`sm` buttons with 4x4 icons are pretty small and probably not super great on mobile, so you might have changed it for that reason? 🤷🏻‍♀️